pkg/types/target.go

```diff
@@ -14,14 +14,14 @@ type Scanner string
 type Scanners []Scanner
 
 const (
-	// VulnTypeUnknown is a vulnerability type of unknown
-	VulnTypeUnknown = VulnType("unknown")
+	// PkgTypeUnknown is a package type of unknown
+	PkgTypeUnknown = PkgType("unknown")
 
-	// VulnTypeOS is a vulnerability type of OS packages
-	VulnTypeOS = VulnType("os")
+	// PkgTypeOS is a package type of OS packages
+	PkgTypeOS = PkgType("os")
 
-	// VulnTypeLibrary is a vulnerability type of programming language dependencies
-	VulnTypeLibrary = VulnType("library")
+	// PkgTypeLibrary is a package type of programming language dependencies
+	PkgTypeLibrary = PkgType("library")
```

Comment:
I want to change it to `lang` or `language` as we lately call them "Language-specific packages". (we used to call them libraries, as you know). But I don't want to break compatibility. Do you think we should open another PR? I think it's a good opportunity.

Reply:
>  I think it's a good opportunity.

Yeah. Now is the best time for this.

> I want to change it to lang or language as we lately call them "Language-specific packages"

I think `lang` will be enough.

> But I don't want to break compatibility. Do you think we should open another PR?

I think it is better to create one more PR for this.
Also we need to create 2 paragraphs in release discussion to attract the attention of users to the fact that we not only renamed flag, but also changed `library` to `lang`.
Also in the new PR we need to add warning when user uses `library` and rename it (like we do with flag aliases)

Reply:
>I think it is better to create one more PR for this.

OK. However, we can announce these changes together in a single thread in GitHub Discussions.
e.g. https://github.com/aquasecurity/trivy/discussions/7010

Reply:
Yes, that's a good idea!